README.md

```diff
@@ -31,14 +30,16 @@ npm takes care of the rest of our dependencies.
     ``` npm install ```
 
 ## Creating a dotenv file
+
 The app needs to point to the hostname where the backend expects to receive HTTP requests. This has to be set before building the client.
 
 In order to set up all that, create a file at the location /var/www/pwa-prototype-reporting/.env . Use the following contents as an example:
```

Comment:
create an env.example file that has the correct default values and instruct people to copy the file into .env (instead of creating one from scract) 

http://ushahidi-backend.io is not a valid URL that will lead them to a real project so let's  also add the following information in the env.example file to ensure folks can execute the code successfully

base_url = https://pwa-prototype.ushahidi.io/
client_secret = '35e7f0bca957836d05ca0492211b0ac707671261'
client_id = 'ushahidiui'
form_id = 1

Reply:
also instead of `create a file at the location /var/www/pwa-prototype-reporting/.env` we should instruct them to 
`Create a new file named ".env" and copy the contents of env.example into it. The .env file has to be placed in the application's root directory (at the same level as env.example).`

Reply:
Btw I'm adding that deployment now, so it won't be available for a few minutes but you can start the changes and I'll ping once it's up

Reply:
Ok, you can now test with that url (https://pwa-prototype.api.ushahidi.io/) . If you go into https://pwa-prototype.ushahidi.io/ and create a user, I will make you admin so you can see the posts too